coding-concepts.js

```diff
@@ -5,69 +5,95 @@
+// // --------------------4) What will this log?
+
+// var onlyOdds = [11, 12, 13, 14, 15].filter(value => value % 2 !== 0)
 // console.log(onlyOdds)
 
-// a) Your answer:
-// b) Verify and explain:
+// // a) Your answer: Once I run this code, I will receive the value of all the odd numbers located in this array only.Since we are using the .filter function to go through the arrray and by using the boolean remainer value %2 ! == 0  to determine if it is equal to 0 making the value even and only returning an odd value
+
+
+
+// // b) Verify and explain:
 
+// As expected I recieved: [ 11, 13, 15 ] as return
+// the function ran through the variable and filtered out the numbers and checked for the odd and even numbers using the %2 !==0 boolean value type.
 
-// --------------------5) What will this log?
+// // --------------------5) What will this log?
 
-var myCodingSkills = {
-  languages: ["JavaScript", "Ruby"],
-  frameworks: ["React", "Ruby on Rails"],
-  databases: "PostgreSQL",
-  versionControl: "GitHub"
-}
+// var myCodingSkills = {
+//   languages: ["JavaScript", "Ruby"],
+//   frameworks: ["React", "Ruby on Rails"],
+//   databases: "PostgreSQL",
+//   versionControl: "GitHub"
+// }
 // console.log(myCodingSkills.languages[0])
 
-// a) Your answer:
-// b) Verify and explain:
+// // a) Your answer: We will get a return of "Javascript". This code is going to access our object, then our languages in our object and then go right to our index of 0 which returns "Javascript" as that is what is in the console log that we are calling back through object destructuring. 
+
+// // b) Verify and explain:
 
+//  As excpected I got a return of "Javascript" since we are just acessing the myCodingSkills object and then the keyvalue property of languages and then the index of 0 in there which is 0. When we console logged it we destructured the object and got access to our exact property through the object.
 
-// --------------------STRETCH: What will this log?
+// // --------------------STRETCH: What will this log?
 
-class Learn {
-  constructor(name){
-    this.student = name,
-    this.cohort = "Bravo",
-    this.year = 2022
-  }
-}
-var learnStudent = new Learn("George")
+// class Learn {
+//   constructor(name){
+//     this.student = name,
+//     this.cohort = "Bravo",
+//     this.year = 2022
+//   }
+// }
+// var learnStudent = new Learn("George")
 // console.log(learnStudent)
 
-// a) Your answer:
-// b) Verify and explain:
+// // a) Your answer: I believe that this will return name: "George", cohort: "Bravo" and year: 2022. I believe this because when a string is declared they dont change unless invovked as an argument like it was done here. 
+
+// // b) Verify and explain: As expected, my answer was correct. The class had an inheritance returned as an argument which in this case was George and overrode name as a constant. Cohort and year were already made constants and name was just modified. 
```

Comment:
To clarify, inheritance is often referred to as a child class that extends from a parent class.  In this case, we do not have any child class.  Name is a parameter that is set inside the constructor and this.student is being assigned to whatever value is passed in as an argument for name.